src/sentry/static/sentry/app/components/charts/releaseSeries.jsx

```diff
@@ -145,13 +146,28 @@ class ReleaseSeries extends React.Component {
   }
 
   setReleasesWithSeries(releases) {
+    const {emphasizeReleases = []} = this.props;
+    const unemphasizedReleases = releases.filter(
+      release => !emphasizeReleases.includes(release.version)
+    );
+    const emphasizedReleases = releases.filter(release =>
+      emphasizeReleases.includes(release.version)
+    );
```

Comment:
[lodash/partition](https://lodash.com/docs/4.17.15#partition) might work well here and would save you a loop.

Reply:
Updated in 962bbe9